ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/CommandDeduplicationIT.scala

```diff
@@ -23,7 +24,9 @@ final class CommandDeduplicationIT(timeoutScaleFactor: Double, ledgerTimeInterva
 
   override def testNamingPrefix: String = "ParticipantCommandDeduplication"
 
-  /** Assertions for append-only schema are important for KV ledgers
-    */
-  override protected def isAppendOnly: Boolean = false
+  override def deduplicationFeatures: CommandDeduplicationBase.DeduplicationFeatures =
```

Comment:
How about renaming this test something like `ParticipantOnlyCommandDeduplicationIT`?

Reply:
I would love to, but this is a default test and I'm not sure about the implications.
No issues about our builds but we would also need to update any external usages (eg canton)